python/src/aiconfig/__init__.py

```diff
@@ -1,14 +1,23 @@
+# Core Data Classes
 from .AIConfigSettings import (
```

Comment:
Can you also rename this to Schema? Or AIConfigSchema?

Reply:
will update in a new diff ontop to schema.py as discussed offline